### PR TITLE
v0.11.0-rc.1

### LINKED ISSUE #3: Scope of Simple Edge Discovery for Fall24 CAMARA release
This issue shall define which issues are within scope of  the API of the Simple Edge Discovery Sub Project for the Fall24 release, and will be regularly updated based on the decision within the EdgeCloud meetings.
 
Resource model name/structure changes
 
* camaraproject/EdgeCloud/issues/226
* camaraproject/EdgeCloud/issues/210
* camaraproject/EdgeCloud/issues/209
* * camaraproject/EdgeCloud/issues/163

Commonalities

* camaraproject/EdgeCloud/issues/225
* camaraproject/EdgeCloud/issues/202
* camaraproject/EdgeCloud/issues/193

Documentation

* camaraproject/EdgeCloud/issues/207

API Release Criteria

* camaraproject/EdgeCloud/issues/213
* camaraproject/EdgeCloud/issues/203

### PR DESCRIPTION
**Pull request by Kevsy**
#### What type of PR is this?

Add one of the following kinds:
* enhancement/feature

#### What this PR does / why we need it:
Initial rc pre-release for Fall Meta Release, with alignment to Commonalities 0.4-rc.1, Identity and Consent Management-0.2.rc1, and Edge Cloud. 

See #3  and/or CHANGLOG.md for full list of fixed issues.

#### Which issue(s) this PR fixes:

<!-- Automatically closes linked issue when PR is merged.

Fixes #3

#### Changelog input

(captured in CHANGELOG.md)

#### Additional documentation 

This section can be blank.

```
docs

```

### CODE REVIEW

**Comment by Kevsy** [comment]
Iinter errors are due to trailing spaces: local lint shows no errors so I'll check what the problem is...

**Comment by Kevsy** [comment]
@javierlozallu  I can't upload the whitespace fix because this is a protected branch, meaning I can't commit to it.

The report above shows *only* trailing whitespace errors for the YAML - so I can fix that when we move from 0.11.0-rc.1 to 0.11.0- rc . We are required too have the rc available by end of Friday, so if you are happy with the rest of this rc1 PR then please approve and I can upload rc with the whitespace fix for approval.

File: code/API_definitions/simple-edge-discovery.yaml [hunk 1]
```diff
@@ -212,26 +265,34 @@ paths:
...
           schema:
             $ref: "#/components/schemas/DeviceIpv6Address"
 
         - name: Network-Access-Identifier
           in: header
+          required: false
           description: |
             3GPP network access identifier for the subscription
             being used by the device.
           example: "4d596ac1-7822-4927-a3c5-d72e1f922c94@domain.com"
```
Comment (javierlozallu) [inline code/API_definitions/simple-edge-discovery.yaml:295]:
```suggestion
```

If I'm not wrong `example` parameter should be removed from NetworkAccessIdentifier object. Let me know if this is correct

**Comment by Kevsy** [comment]
Thanks @javierlozallu , well spotted - I will create a PR for this PR(!) because it is a protected branch so I can not apply new commits.

File: documentation/API_documentation/SED_User_Story.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+Note:
+- the Edge Cloud Zone platform may be hosted by the operator or a 3rd party hyperscaler (in which case the developer will need an account with that hyperscaler to create instances)
+
+
+```mermaid
+sequenceDiagram
+    Note over Application Client,Operator: PRE App on device attached <br/> to operator network
+    Note over Application Server,Operator: PRE App developer registered, <br/> authenticated and authorised
+    Application Client->>Operator: GET /edge-cloud-zones?filter=closest {header: device identifier(s)}
+    Operator->>Operator: calculate closest Edge Cloud Xone for device
```
Comment (javierlozallu) [inline documentation/API_documentation/SED_User_Story.md:31]:
```suggestion
    Operator->>Operator: calculate closest Edge Cloud Zone for device
```

**Review by javierlozallu — DISMISSED** [review]

**Review by javierlozallu — APPROVED** [review]

**Pull request merged**